packages/www/hooks/use-api.tsx

```diff
@@ -530,14 +530,30 @@ const makeContext = (state: ApiState, setState) => {
       return [streams, nextCursor, res];
     },
 
-    async getStreams(userId): Promise<Array<Stream>> {
+    async getStreams(
+      userId: string,
+      opts?: {
+        filters: Array<{ id: string; value: string }>;
+        limit: string;
+        cursor: string;
+        order: string;
+      }
+    ): Promise<[Stream[], string]> {
+      const filters = opts?.filters ? JSON.stringify(opts?.filters) : undefined;
       const [res, streams] = await context.fetch(
-        `/stream?userId=${userId}&streamsonly=1`
+        `/stream?${qs.stringify({
+          userId,
+          filters,
+          order: opts?.order,
+          limit: opts?.limit,
+          cursor: opts?.cursor,
```

Comment:
Add `streamsonly: 1` as a parameter. This ensures we only return streams. Otherwise, it will return both streams and sessions.